assets/performanceprofile/scripts/cpuset-configure.sh

```diff
@@ -36,3 +39,25 @@ mkdir -p "$machine"
 
 # It's unlikely, but possible, that this cpuset already existed. Iterate just in case.
 for file in $(find "$machine" -name cpuset.cpus | sort -r); do echo "$reserved_set" > "$file"; done
+
+# OVS is running in its own slice that spans all cpus. The real affinity is managed by OVN-K ovnkube-node daemonset
+# Make sure this slice will not enable cpu balancing for other slice configured by this script.
+# This might seem counter-intuitive, but this will actually NOT disable cpu balancing for OVS itself.
+# - OVS has access to reserved cpus, but those have balancing enabled via the `system` cgroup created above
+# - OVS has access to isolated cpus that are currently not assigned to pinned pods. Those have balancing enabled by the
```

Comment:
What about the case when workload partition enabled? 
Will it affine OVS to reserved?

Reply:
Yes, the ovnkube-node pod will only run on reserved and pass that affinity to OVS too.

Reply:
Once, at cluster startup time. Then it will never change.